Organic/Affiliate.php

```diff
@@ -58,4 +59,17 @@ public function register_gutenberg_block() {
             plugin_dir_path( __DIR__ ) . 'affiliate/blocks/productCarousel'
         );
     }
+
+    public function admin_post_page_enqueue( $hook_suffix ) {
+        // Scripts to enqueue only for WP Post pages.
+        if ( 'post.php' != $hook_suffix && 'post-new.php' != $hook_suffix ) {
+            return;
+        }
+        wp_enqueue_script(
+            'on-post-load-scripts',
+            plugins_url( 'affiliate/initSDKOnPostLoad.js', __DIR__ ),
+            [ 'organic-sdk' ],
+            $this->organic->version
+        );
```

Comment:
The code looks functional to me, I have a couple of suggestions tho:

In my experience using if statements that return truthy values instead of falsy values perhaps makes the code more readable and self-explanatory, it helps make clear what you expect to be true. 

Also, I would suggest the use of `===` instead of `==` which can prevent unexpected behavior and bugs caused by type conversion as a general practice. What do you think? @Mwindo 

```php
if ( 'post.php' === $hook_suffix && 'post-new.php' === $hook_suffix ) {
  wp_enqueue_script(
      'on-post-load-scripts',
      plugins_url( 'affiliate/initSDKOnPostLoad.js', __DIR__ ),
      [ 'organic-sdk' ],
      $this->organic->version
  );
}
```

Reply:
Yes, this is better. I'm not sure why I didn't write it this way originally. Thank you!